index.js

```diff
@@ -35,11 +35,11 @@ clim.logWrite = function(level, prefixes, msg) {
   level = (f[level.toLowerCase()] || function(p) {return p;})(level);
   var line = util.format("%s [%s] %s", pfx, level, msg);
   switch (level) {
-    case "ERROR":
-      process.stderr.write(line + "\n");
-      break;
-    default:
-      process.stdout.write(line + "\n");
+  case "ERROR":
+    process.stderr.write(line + "\n");
+    break;
+  default:
+    process.stdout.write(line + "\n");
```

Comment:
Ah, this was autofixed. If you prefer indented case style, I think that's a flag on the ESLint `indent` rule that I can add and auto-fix again to put us back to previous.